### PR TITLE
[MM-44860] Screen reader announcing link copied on copying the link in invite modal

### PR DESCRIPTION
**Pull request by babinderrathi**
#### Summary
Screen reader announcing link copied on copying the link in invite modal

#### Ticket Link

  Fixes https://mattermost.atlassian.net/browse/MM-44860

#### Related Pull Requests
None

#### Screenshots

|  before  |
|----|
https://user-images.githubusercontent.com/102028591/188860492-8e308cf1-e6e7-42bb-a06c-65cfa5809290.mp4

|  after |
|----|
https://user-images.githubusercontent.com/102028591/188861546-eed97466-0b40-440d-85b7-fd27cec8ac99.mp4

#### Release Note
```release-note
* Screen reader announcing link copied on copieng the link in invite modal
```

### CODE REVIEW

File: components/invitation_modal/invite_view.tsx [hunk 1]
```diff
@@ -86,31 +87,37 @@ export default function InviteView(props: Props) {
         text: inviteURL,
     });
 
-    const copyButton = (
+    const copyButton = copyText.copiedRecently ? (
         <button
             onClick={copyText.onClick}
             data-testid='InviteView__copyInviteLink'
-            aria-label='team invite link'
+            aria-label={localizeMessage(
```
Comment (hmhealey) [inline components/invitation_modal/invite_view.tsx:94]: Instead of `localizeMessage` could you instead use `formatMessage` for these?

**Review by hmhealey — CHANGES_REQUESTED** [review]
Thanks for the PR, @babinderrathi! There's one small thing I'd like changed, but this looks good to me.

One other thing I did notice is that I would've expected the button to need an aria-live so that changes in the label are read out, but I'm not too familiar with screen readers, so perhaps that isn't needed as long as the button is stays in focus after interacting with it

**Review by ghost — APPROVED** [review]
Thanks @babinderrathi and also for adding the translations, LGTM!

**Review by hmhealey — APPROVED** [review]
Thanks for changing that!

**Review by mkraft — APPROVED** [review]

**Comment by AshishDhama** [comment]
@jgilliam17 this is ready for QA Review

**Comment by jgilliam17** [comment]
Thanks @babinderrathi 
macOS VoiceOver is announcing "Link Copied", but Windows Jaws reader is still not announcing when the link is copied. Instead it announces "space" which is the key pressed to activate/copy the link.

**Comment by hmhealey** [comment]
Hi @babinderrathi. Are you able to look into that issue with JAWS? It's generally the screen reader that we test with since we have some big deployments using it.

If you need help fixing it or reproducing the issue yourself, let us know and we can help out.

**Review by mkraft — APPROVED** [review]

**Review by ghost — APPROVED** [review]
LGTM!

**Comment by AshishDhama** [comment]
@jgilliam17 could you please check if this working as expected now.

**Comment by jgilliam17** [comment]
@AshishDhama JAWS reader is announcing the whole Invite link e.g. https://mattermost-webapp-pr-11081.test.mattermost.cloud/signup_user_complete/?id=p561t9zowb8j5ca3kc8goufsme&sbr=fa
Instead of only announcing "Link Copied" or similar

**Comment by AshishDhama** [comment]
@babinderrathi could you please fix this?

**Comment by lindy65** [comment]
Hey @AshishDhama - are we still having issues setting up spinwicks?

**Comment by AshishDhama** [comment]
@lindy65 Yeah, for fork pr, there are some issues

**Comment by lindy65** [comment]
Thanks @AshishDhama - I will test this as soon as the spinwick issues are resolved.

**Comment by AshishDhama** [comment]
@lindy65 You can test this pr now.

**Review by lindy65 — APPROVED** [review]
Thanks @babinderrathi - this is fixed, Jaws announces that the link is copied

cc @AshishDhama

**Comment by M-ZubairAhmed** [comment]
Hey there!, Thank you for your PR.
Webapp is now part of mattermost monorepo now at https://github.com/mattermost/mattermost-server/tree/master/webapp Would you be interested in opening the PR over there?

**Comment by hmhealey** [comment]
The web app now lives in the `webapp/channels` over in https://github.com/mattermost/mattermost-server. We decided to move it to be a monorepo to make both development and our CI pipelines easier, but it's unfortunately disruptive right now. We've updated [our setup docs](https://developers.mattermost.com/contribute/developer-setup/) and added [some steps for migrating to working with the monorepo](https://developers.mattermost.com/contribute/monorepo-migration-notes/).

Copying the changes over there hopefully shouldn't be too hard. There's a few changes to file paths (such as the code living in a dedicated `src` subdirectory), but most things should look the same as they do now. 

Let us know if you could use any help with this. Hopefully the process is smooth.

**Comment by AshishDhama** [comment]
Closing in favour of https://github.com/mattermost/mattermost-server/pull/22785

**Pull request closed without merging**